spec.md

```diff
@@ -0,0 +1,115 @@
+# Specification
+
+Implement in your favorate languages by following this specification.
+
+* Date: 2015-09-07
+* Authors: Hsiaoming Yang, Jiangge Zhang
+
+
+## Interface
+
+Namespace should always be **GB2260**.
+
+## Data Structure
+
+There is a standard structure named `Division`. Those following properties
+should be included in it.
+
+- `code`: The six-digit number of the specific administrative division.
```

Comment:
I suggest add data type to field descriptions, especially the `code` field, because `code` usually means `int`.